src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -456,7 +456,7 @@ export default {
 	the margin applied to the content is added to the total modal width,
 	so here we subtract it to the width and height of the content.
```

Comment:
Old comment, that needs to be removed: https://github.com/nextcloud/spreed/commit/c65dc3847e00242d213a5db5861ff2ad724d9bac